pkg/kubelet/kubeletconfig/configfiles/configfiles.go

```diff
@@ -83,8 +84,18 @@ func resolveRelativePaths(paths []*string, root string) {
 	for _, path := range paths {
 		// leave empty paths alone, "no path" is a valid input
 		// do not attempt to resolve paths that are already absolute
-		if len(*path) > 0 && !filepath.IsAbs(*path) {
+		if len(*path) > 0 && !isAbs(*path) {
 			*path = filepath.Join(root, *path)
 		}
 	}
 }
+
+func isAbs(path string) bool {
+	if filepath.IsAbs(path) {
+		return true
+	}
+
+	// on Windows, filepath.IsAbs will not return True for paths prefixed with a slash, even
+	// though they can be used as absolute paths.
+	return goruntime.GOOS == "windows" && len(path) > 0 && (path[0] == '\\' || path[0] == '/')
```

Comment:
Is this approach necessary?  [src/path/filepath/path_windows.go#L40](https://github.com/golang/go/blob/master/src/path/filepath/path_windows.go#L49)

Can you elaborate in which situation windows path with "/" or "" considered as absolute paths?

Reply:
Sure. At the link you've given, it checks if it starts with ``\\`` or ``//`` exactly, but not for simply ``\`` or ``/``. ``filepath.IsAbs`` does not consider those as absolute paths, even though Windows does: https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats